File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/javadoc/comment/direct/Java/C/s780275612_commented.c
#include <stdio.h>
#include <math.h>

int main() {
    int intA;
    // Input an integer from the user
    scanf("%d", &intA);

    // Calculate the cube of the input number
    double dbAncer = pow(intA, 3);
    int intAncer = (int)dbAncer;
    printf("%d\n", intAncer);
    
    return 0; // End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/mad/Java/Go/s780275612_commented.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var intA int
	fmt.Scan(&intA)
	intAncer := int(math.Pow(float64(intA), 3))
	fmt.Println(intAncer)
}
// End of Code
